notebooks/en/analyzing_art_with_hf_and_fiftyone.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>I think multimodal is a very broad term here, the word you're looking for is image-text encoder models IMO (SigLIP, EvaCLIP etc) you could say CLIP-like models</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>